src/jupyter-aou/docker-compose.yaml

```diff
@@ -0,0 +1,50 @@
+version: '2.4'
+services:
+  app:
+    container_name: "application-server"
+    image: "gcr.io/deeplearning-platform-release/tf-gpu.2-11.py310"
+    restart: always
+    volumes:
+      - .:/workspace:cached
+      - work:/home/jovyan/work
+      - gcsfuse-mount:/mnt/gcs
+    ports:
+      - '8888:8888'
+    environment:
+      JUPYTER_ENABLE_LAB: "yes"
+    command: "start-notebook.py --ServerApp.token='' --ServerApp.password='' --ServerApp.root_dir=/home/jovyan/work --ServerApp.allow_origin='*'"
+    networks:
+      - app-network
+    devices:
+      - /dev/fuse
+    security_opt:
+      - apparmor:unconfined
+  wondershaper:
+    container_name: "wondershaper"
+    image: "us.gcr.io/broad-dsp-gcr-public/wondershaper:0.0.1"
+    restart: always
+    network_mode: "host"
+    cap_add:
+      - NET_ADMIN
+  gcsfuse:
+    container_name: "gcsfuse"
+    image: "google/cloud-sdk:latest"
+    restart: always
+    volumes:
+      - gcsfuse-mount:/mnt/gcs
+    entrypoint: >
+      /bin/sh -c "
+      gcsfuse YOUR_BUCKET_NAME /mnt/gcs
```

Comment:
todo - pass YOUR_BUCKET_NAME as env var from Mikey vm init script

Reply:
This is a cool exploration. I think the workbench way is to use the workbench CLI and run `wb resource mount`, which mounts all resource buckets in the workspace. This means we may need to build another image that specifically does this. The other problem is that now that users cannot remount buckets themselves, we'll need a cron in the mount sidecar that tries to remount every hour or so.